content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -78,7 +77,7 @@ for creating custom clusters.
 Kubeadm executes a set of preflight checks before starting the init, with the aim to verify preconditions and avoid common cluster startup problems.
 In any case the user can skip specific preflight checks (or eventually all preflight checks) with the `--ignore-preflight-errors` option.
 
-- [warning] If the Kubernetes version to use (specified with the `--kubernetes-version` flag) is at least one minor version higher than the kubeadm CLI version
+- [warning] If the Kubernetes version to use (specified with the `--kubernetes-version` flag) is at least one minor version higher than the kubeadm CLI version.
```

Comment:
What about using the shortcode here? See https://kubernetes.io/docs/home/contribute/style-guide/#shortcodes

I know this is outside of the scope of your change...

Reply:
This isn't a warning, it's a listing of a type of error message. I thought the same thing and almost changed it. I forgot to revert the period, that's it.

Reply:
Oh - indeed...

Reply:
@mistyhacks.  Agreed! Good catch